rwmutex_test.go

```diff
@@ -0,0 +1,169 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// GOMAXPROCS=10 go test
+
+package debugsync
+
+import (
+	"fmt"
+	"runtime"
+	"sync/atomic"
+	"testing"
+)
+
+// There is a modified copy of this file in runtime/rwmutex_test.go.
+// If you make any changes here, see if you should make them there.
```

Comment:
Huh? I assume this is left over from something older, right?
I don't think anybody maintaining this code is going to make changes to the Go `runtime` standard library 😁

Reply:
I'll remove that 😄